champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/axe/WolfsFury.java

```diff
@@ -114,6 +114,10 @@ private boolean expire(Player player, boolean force) {
         if (player == null) {
             return true;
         }
+        if (!isHolding(player)) {
+            expire(player, true);
+            deactivate(player);
+        }
```

Comment:
i mean just have it as

`
 if (!isHolding(player)) {
          deactivate(player);
          return true;
 }
`